Libraries/LibELF/AuxiliaryVector.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2020, The SerenityOS developers.
```

Comment:
I notice you are using the generic copyright here while you have a personal one in other parts of the project.
Either way is fine of course, just pointing out the deviation :)

Reply:
Yeah, was intentional here.